packages/cli/src/commands/discovery/init.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'node:fs';
+import degit from 'degit';
+import { Command } from '@oclif/core';
+import { confirm } from '@inquirer/prompts'
+
+export default class Init extends Command {
+  static description = "Creates a discovery folder based on the starter.store template.";
+
+  static examples = ["$ yarn faststore discovery init"];
```

Comment:
@gvc ok cool! I think I had it confused because the [ticket](https://vtex-dev.atlassian.net/jira/software/c/projects/SFS/boards/1054?selectedIssue=SFS-1073) implied that `fsp discovery init store-a/discovery` should be the de-facto command to be run, but taking in consideration what will be the final product of this PR, your suggestion makes more sense.